<launch>
  <!-- planner -->
  <param name="/irl_features/max_msg_age" value="0.1" type="double"/>
  
  <param name="/irl_features/feature_type" value="helbing"/>
  <param name="/irl_features/feature_params" value="{ 'radius': 2}"/>
  <param name="/irl_features/weights" value="[0.1, 0.2, 0.3, 0.4]"/>

  <!-- grid dimension -->
  <param name="/irl_features/cell_size" value="1.0" type="double"/>
  <param name="/irl_features/x1" value="-20.0" type="double"/>
  <param name="/irl_features/y1" value="-10.0" type="double"/>
  <param name="/irl_features/x2" value="100.0" type="double"/>
  <param name="/irl_features/y2" value="10.0" type="double"/>

  <param name="/irl_features/speed" value="2.8" type="double"/>

  <node name="irl_features" pkg="belly" type="irl_features.py" />
</launch>
